lib/Cleantalk/Antispam/Integrations/OtterBlocksForm.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Cleantalk\Antispam\Integrations;
+
+use Cleantalk\ApbctWP\Variables\Cookie;
+
+class OtterBlocksForm extends IntegrationBase
+{
+    private $form_data_request;
+
+    /**
+     * @inheritDoc
+     */
+    public function getDataForChecking($argument)
+    {
+        $this->form_data_request = $argument;
```

Comment:
Validate that the $argument is instance of the class expected and if the class has such methods (set_error & get_fields).

If this validation ignored there will be fatal errors if the plugin suddenly updated with the class changes.